non-free.md

```diff
@@ -102,6 +103,13 @@
 - [Yetishare](https://yetishare.com) - Powerful file hosting script with support for Amazon S3, Wasabi, Backblaze, local, direct and SFTP storage. ([Demo](https://fhscript.com)) `⊘ Proprietary` `PHP`
 
 
+### File Transfer - Object Storage & File Servers
+
+**[`^        back to top        ^`](#awesome-selfhosted)**
+
+- [DaneeCloud](https://docs.daneeskripter.dev/daneecloud/) - DaneeCloud is web application, that works like storage hosting. It includes user management, file management, video and audio player, roles (admin, user, ...), email authentication, recaptcha, file sharing and other features. ([Demo](https://cloud.daneeskripter.dev), [Source Code](https://github.com/DaneeSkripter/DaneeCloud)) `GPL-3.0` `Nodejs/JavaScript/EJS`
```

Comment:
```suggestion
- [DaneeCloud](https://docs.daneeskripter.dev/daneecloud/) - Multi-user file storage and sharing application. ([Demo](https://cloud.daneeskripter.dev), [Source Code](https://github.com/DaneeSkripter/DaneeCloud)) `GPL-3.0/SSPL-1.0` `Nodejs/JavaScript/EJS`
```

Edited the license to reflect that it requires MongoDB which is under SSPL-1.0 (this is the reason why it was moved to non-free). Also cleaner description.